cibuildwheel/util.py

```diff
@@ -127,13 +128,32 @@ def call(
     args_ = [str(arg) for arg in args]
     # print the command executing for the logs
     print("+ " + " ".join(shlex.quote(a) for a in args_))
-    kwargs: dict[str, Any] = {}
-    if capture_stdout:
-        kwargs["universal_newlines"] = True
-        kwargs["stdout"] = subprocess.PIPE
-    result = subprocess.run(args_, check=True, shell=IS_WIN, env=env, cwd=cwd, **kwargs)
+    # workaround platform behaviour differences outlined
+    # in https://github.com/python/cpython/issues/52803
+    path_env = env if env is not None else os.environ
+    path = path_env.get("PATH", None)
+    executable = shutil.which(args_[0], path=path)
+    if executable is None:
+        msg = f"Couldn't find {args_[0]!r} in PATH {path!r}"
+        raise FatalError(msg)
+    args_[0] = executable
```

Comment:
Given shell is still used on Windows, this could just be removed.
The upside if keeping this is that when a CalledProcessError is raised, we get the full path to the executable rather than just the executable name.